### PR TITLE
API-1537: Bump k8s 1.26.2 proof

### PR DESCRIPTION
**Pull request by dinhxuanvu**
Proof PR for bumping k8s 1.26.2

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,15 +27,15 @@ require (
 	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
 	go.etcd.io/etcd/client/v3 v3.5.5
-	k8s.io/api v0.26.1
+	k8s.io/api v0.26.2
 	k8s.io/apiextensions-apiserver v0.26.1
```
Comment (dgrisonnet) [inline go.mod:31]: this needs to be upgraded

File: go.mod [hunk 2]
```diff
@@ -27,15 +27,15 @@ require (
...
-	k8s.io/component-helpers v0.26.1
+	k8s.io/apimachinery v0.26.2
+	k8s.io/apiserver v0.26.2
+	k8s.io/client-go v0.26.2
+	k8s.io/cloud-provider v0.26.2
+	k8s.io/code-generator v0.26.2
+	k8s.io/component-base v0.26.2
+	k8s.io/component-helpers v0.26.2
 	k8s.io/klog/v2 v2.80.1
 	k8s.io/kube-aggregator v0.26.1
```
Comment (dgrisonnet) [inline go.mod:40]: this one also needs to be updated.

We should also update the two following ones:
```
	k8s.io/kubectl v0.26.1
	k8s.io/kubernetes v1.26.1
```

**Review by sanchezl — APPROVED** [review]
/lgtm

**Pull request merged**